### PR TITLE
Icon integration

### PR DESCRIPTION
**Pull request by ianmcburnie**
## Description

Integrates the new background icon mixins into breadcrumbs, pagination, dialog, combobox and menu. The only thing not integrated is select. Sean is going to look into that one, but it can wait if necessary.  

## Context

All background icons should come from the mixins which hold the single source of truth for the base64 encoding.

## Screenshots

screen shot 2018-04-19 at 2 57 54 pm

screen shot 2018-04-19 at 10 48 12 am

screen shot 2018-04-19 at 9 43 40 pm

screen shot 2018-04-19 at 4 09 13 pm

screen shot 2018-04-19 at 4 09 07 pm

The horizontal alignment of those last two look a little off (the expand icon is not perfectly aligned with tick icon), but I believe they've always been that way. We should fix as part of #79.

### CODE REVIEW

**Comment by seangates** [comment]
Was inches away from asking about the disabled button icon ... and then you fixed it.

**Comment by seangates** [comment]
Should we also change this, as we have an chevron-up now?

https://github.com/eBay/skin/blob/866b98a8832f9b5d057a7db22c42e74b52a19639/src/less/button/ds6/button-base.less#L127

**Comment by ianmcburnie** [comment]
Is it using the bold version currently? It doesn't seem so. I'm just trying to do a like for like replacement in this PR in order not to break anything. I believe there was a separate task for moving to bolder versions?

**Comment by seangates** [comment]
It is not using the bold version currently, and yes, there is a story for it: #79 

But, since you're there, and you're changing it anyway, you might as well fix it. You might as well be breaking it this way! 😉 😂 

As per the chevron-up ... we could fix that later, which is why I didn't request it as a change. ;-)

**Comment by seangates** [comment]
Wait ... this already got changed earlier. Never mind. I was mixing two changes in my mind.

Carry on.

**Comment by ianmcburnie** [comment]
I see what you mean, will change to bold icon.

**Comment by ianmcburnie** [comment]
Actually, I won't. Because I need to create a chevron down disabled base64 too. I just want to get this PR in so I can move on with the stylehint stuff.

Also, when looking at the rotate it's made me realize that we might need a version of the mixins without the width and height, otherwise width and height gets applied again even though we only want to change the image.

File: dist/button/ds6/button.css [hunk 1]
```diff
@@ -104,7 +104,9 @@ button.expand-btn[aria-expanded="true"] .expand-btn__icon {
...
+  background-image: url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMSIgaGVpZ2h0PSIxMiIgZmlsbD0iI2M3YzdjNyI+PHBhdGggZD0iTTE5LjAwNi4zNTFMMTAuNSA5LjEwNCAxLjk5Mi4zNWExLjE0OSAxLjE0OSAwIDAgMC0xLjY1MSAwQTEuMjI2IDEuMjI2IDAgMCAwIDAgMS4xNTJ2LjA5NmMuMDEyLjI5MS4xMjYuNTc5LjM0LjgwMWw5LjMzNSA5LjZjLjIyLjIyNy41MDguMzQ0Ljc5Ni4zNTFoLjA1N2MuMjg4LS4wMDguNTc2LS4xMjQuNzk3LS4zNTFsOS4zMzMtOS42YTEuMjEgMS4yMSAwIDAgMCAuMzQyLS44MXYtLjA3OGExLjIxIDEuMjEgMCAwIDAtLjM0Mi0uODEgMS4xNDggMS4xNDggMCAwIDAtMS42NTIgMHoiLz48L3N2Zz4=');
+  height: 100%;
```
Comment (ianmcburnie) [inline dist/button/ds6/button.css]: This is an example of where the current mixing setup causes width and height to be reapplied. I may look into refactoring them again slightly.

Comment (seangates) [reply dist/button/ds6/button.css]: 👍

**Pull request merged**